confirm_modal.js

```diff
@@ -0,0 +1,27 @@
+(function(){
+    function confirm_modal(text, options={}){
+        const {modal_type="default", header="Attention!"} = options;
+        if(!text) throw new Error("Modal text is required!");
+        const modal_wrapper = document.createElement("div");
+        modal_wrapper.className = "confirm_modal_wrapper";
```

Comment:
Maybe we need to create common class "beauty_modal_wrapper" for both modal types.